FEEDBACK.md

```diff
@@ -0,0 +1,334 @@
+
+```javascript
+bucket.init({
+  feedback: {
+    automaticPrompting: true,
+    promptHandler: (promptMessage, handlers) => {
+      // Pass your overrides here. Everything is optional
+      handlers.openFeedbackForm({
+        title: promptMessage.question,
+
+        position: POSITION_CONFIG, // See positioning section
+        translations: TRANSLATION_KEYS, // See internationalization section
+
+        // Trigger side effects with the collected data,
+        // for example posting it back into your own CRM
+        onAfterSubmit: (feedback) => {
+          storeFeedbackInCRM({
+            score: feedback.score,
+            comment: feedback.comment,
+          });
+        },
+      });
+    },
+  },
+});
+```
+
+See also:
+
+- [Positioning and behavior](#positioning-and-behavior) for the position option.
+- [Runtime language configuration](#runtime-language-configuration) if you want to translate the feedback UI.
+- [Use your own UI to collect feedback](#using-your-own-ui-to-collect-feedback) if the feedback UI doesn't match your design.
+
+## Positioning and behavior
+
+The feedback UI can be configured to be placed and behave in 3 different ways:
+
+### Positioning configuration
+
+#### Modal
+
+A modal overlay with a backdrop that blocks interaction with the remaining page. Can be dismissed with `<ESC>` or a close button. It is always centered on the page, making it the primary thing the user needs to focus on.
+
+Using a modal is the strongest possible push for feedback. You are interrupting the users normal flow, which can cause annoyance. A good use case fora modal is when the user finishes a linear flow that they don"t use often, for example setting up an account.
+
+```javascript
+position: {
+  type: "MODAL";
+}
+```
+
+#### Dialog
+
+A dialog that appears in a corner of the viewport, but lets the user continue with their interaction with the page. Can be dismissed with a close button, but also automatically disappears after a relatively shot time period if the user does not interact with it.
+
+Using a dialog is a soft push for feedback. It lets the user continue with their work with a minimal amount of intrusion. The user can opt in to responding, but is not required to. A good use case for this behavior is when a user used a feature where the expected outcome is predictable, possibly because they have used it multiple times before. For example: Uploading a file, switching to a different view of a visualization, visiting a specific page or manipulating some data.
+
+The default feedback UI behavior is a dialog placed at the bottom right of the users viewport.
+
+```javascript
+position: {
+  type: "DIALOG",
+  placement: "top-left" | "top-right" | "bottom-left" | "bottom-right"
+}
+```
+
+#### Popover
+
+A popover that is anchored relative to a DOM-element (typically a button). Can be dismissed with an outside click or pressing a close button.
```

Comment:
I suggest:

A popover that is anchored relative to a DOM-element (typically a button). It can be dismissed by clicking outside the popover or by pressing the dedicated close button.

Reply:
```suggestion
A popover that is anchored relative to a DOM-element (typically a button). It can be dismissed by clicking outside the popover or by pressing the dedicated close button.
```